services/web/server/src/simcore_service_webserver/conversations/_controller/_conversations_messages_rest.py

```diff
@@ -70,27 +80,70 @@ async def create_conversation_message(request: web.Request):
     conversation = await _conversation_service.get_conversation_for_user(
         app=request.app,
         conversation_id=path_params.conversation_id,
-        user_group_id=user_primary_gid,
+        user_group_id=user["primary_gid"],
     )
     # Ensure only support conversations are allowed
     if conversation.type != ConversationType.SUPPORT:
         raise_unsupported_type(conversation.type)
 
-    message = await _conversation_message_service.create_message(
-        app=request.app,
-        user_id=req_ctx.user_id,
-        project_id=None,  # Support conversations don't use project_id
-        conversation_id=path_params.conversation_id,
-        content=body_params.content,
-        type_=body_params.type,
+    message, is_first_message = (
+        await _conversation_message_service.create_support_message_and_check_if_it_is_first_message(
+            app=request.app,
+            user_id=req_ctx.user_id,
+            project_id=None,  # Support conversations don't use project_id
+            conversation_id=path_params.conversation_id,
+            content=body_params.content,
+            type_=body_params.type,
+        )
     )
 
+    # NOTE: This is done here in the Controller layer, as the interface around email currently needs request
+    if is_first_message:
+        try:
+            product = products_web.get_current_product(request)
+            template_name = "request_support.jinja2"
+            destination_email = product.support_email
+            email_template_path = await products_web.get_product_template_path(
+                request, template_name
+            )
+            _url = request.url
+            if _url.port:
+                _conversation_url = f"{_url.scheme}://{_url.host}:{_url.port}/#/conversations/{path_params.conversation_id}"
+            else:
+                _conversation_url = f"{_url.scheme}://{_url.host}/#/conversations/{path_params.conversation_id}"
+            _extra_context = conversation.extra_context
+            await email_service.send_email_from_template(
+                request,
+                from_=product.support_email,
+                to=destination_email,
+                template=email_template_path,
+                context={
+                    "host": request.host,
+                    "product": product.model_dump(
+                        include={
+                            "display_name",
+                        }
+                    ),
+                    "first_name": user["first_name"],
+                    "last_name": user["last_name"],
+                    "user_email": user["email"],
+                    "conversation_url": _conversation_url,
+                    "message_content": message.content,
+                    "extra_context": _extra_context,
+                    "dumps": functools.partial(_json_encoder_and_dumps, indent=1),
+                },
+            )
+        except Exception:  # pylint: disable=broad-except
```

Comment:
why not use `with log_catch` here instead? it also prints the exception and you can set to `reraise=False`.